File: Day8/day8.py
"""implementing day8 in python"""

if __name__ == "__main__":
    T = int(input())

    contacts = {}

    for _ in range(0, T):
        contact = list(input().split(" "))
        contact_infos = [(contact[0], contact[1])]
        contacts.update(contact_infos)

    for _ in range(0, T):
        contact_name = input()
        if contacts.get(contact_name):
            print(contacts.get(contact_name))
        else:
            print("Not found")

File: Day8/day8.java
import java.io.*;
import java.util.*;

public class Solution {

  public static void main(String[] args) {
    Scanner in = new Scanner(System.in);
    int t = in.nextInt();
    in.nextLine();
    HashMap<String, String> contacts = new HashMap<String, String>();

    for (int i = 0; i < t; i++) {
      String[] contact = in.nextLine().split(" ", 2);
      contacts.put(contact[0], contact[1]);
    }

    for (int i = 0; i < t; i++) {
      String contact_name = in.nextLine();
      if (contacts.containsKey(contact_name))
        System.out.println(contact_name + "=" + contacts.get(contact_name));
      else
        System.out.println("Not found");
    }
  }
}
